Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
@xu-cheng Sure, If you're happy with that. You took my idea and condensed it into code that isn't actually hideous to look at, so I'm content to roll with your code suggestions here. The new code block functions as you'd expect it to, so no objections here.